filebeat/beater/filebeat.go

```diff
@@ -56,46 +55,37 @@ func (fb *Filebeat) Run(b *beat.Beat) error {
 
 	var err error
 
-	// Init channels
-	fb.publisherChan = make(chan []*input.FileEvent, 1)
+	// Channel from harvesters to spooler
+	publisherChan := make(chan []*input.FileEvent, 1)
 
 	// Setup registrar to persist state
-	fb.registrar, err = crawler.NewRegistrar(fb.FbConfig.Filebeat.RegistryFile)
+	registrar, err := registrar.NewRegistrar(fb.FbConfig.Filebeat.RegistryFile)
```

Comment:
will change all accordingly. Now that each is int its own package this makes much more sense.